src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,25 +34,33 @@
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${currentTodo.id}`}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-            <button type="button" className="delete" data-cy="modal-close" />
+            <button
+              type="button"
+              className="delete"
+              data-cy="modal-close"
+              onClick={() => onCurrentTodo(null)}
```

Comment:
The method name 'onCurrentTodo' doesn't follow the naming conventions for methods. Consider renaming it to something like 'handleCurrentTodo'. Here are some naming conventions for methods: https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364